openff/evaluator/utils/utils.py

```diff
@@ -258,3 +258,7 @@ def has_openeye():
         available = False
 
     return available
+
+
+def isfile_notempty(path):
```

Comment:
I think it would be good to make the name a bit more verbose:
```suggestion
def is_file_and_not_empty(path):
```

Reply:
It would also be good to expose this in `openff.evaluator.utils` by adding an entry to the `__init__.py`.

Reply:
Finally can you add a unit test for this in `tests.test_utilts.test_utilts`.

Reply:
Done! Thanks @SimonBoothroyd!